src/metabase/driver/druid/query_processor.clj

```diff
@@ -217,6 +218,16 @@
 
 (defn- ag:filtered  [filtr aggregator] {:type :filtered, :filter filtr, :aggregator aggregator})
 
+(defn- ag:distinct [field output-name]
+  (if (= (:base-type field) :type/DruidHyperUnique)
```

Comment:
Doesn't matter right now in the codebase, but it seems like this should be `isa?` for the future.